rasa/utils/tensorflow/temp_keras_modules.py

```diff
@@ -279,7 +279,6 @@ def fit(
             ValueError: In case of mismatch between the provided input data
                 and what the model expects or when the input data is empty.
         """
-        base_layer.keras_api_gauge.get_cell("fit").set(True)
```

Comment:
this change should be reverted because we mimic the keras 2.7 version here and that contains that: https://github.com/keras-team/keras/blob/v2.7.0/keras/engine/training.py#L1131